server/services/user.service.js

```diff
@@ -74,12 +90,11 @@ export const login = async userParam => {
   const { email, password } = userParam;
 
   // Validation
-  let error = {};
-  let user = {};
+  const user = {};
   let userLoggedIn;
-  let dataToReturn = {};
+  const dataToReturn = {};
```

Comment:
This can probably be declared below where it's used

Reply:
Fixed